code/game/gamemodes/blob/theblob.dm

```diff
@@ -25,13 +25,18 @@
 	src.dir = pick(1, 2, 4, 8)
 	src.update_icon()
 	..(loc)
-	for(var/atom/A in loc)
-		A.blob_act()
+	ConsumeTile()
 	return
 
 /obj/effect/blob/proc/creation_action() //When it's created by the overmind, do this.
 	return
 
+/obj/effect/blob/update_icon() //Blobs use update icon for health checks

```

Comment:
todo: fix this, use update_icon as it is meant to be used(as in not for health checks)